starboard/linux/shared/configuration_constants.cc

```diff
@@ -136,3 +136,7 @@ const uint32_t kSbUserMaxSignedIn = 1;
 // The maximum size the cache directory is allowed to use in bytes.
 const uint32_t kSbMaxSystemPathCacheDirectorySize = 24 << 20;  // 24MiB
 #endif
+
+#if SB_API_VERSION >= 16
+SB_EXPORT extern const bool kHasPartialAudioFramesSupport = false;
```

Comment:
I think it's worth to default this to true, and if partner is failing NPLB test, they will come back and fix this.

Reply:
Change around